crates/telio-firewall/src/firewall.rs

```diff
@@ -1260,6 +1260,11 @@ pub mod tests {
             assert_eq!(fw.process_outbound_packet(&make_peer(), &make_tcp(us, them, TcpFlags::FIN)), true);
             assert_eq!(fw.tcp.lock().unwrap().len(), 1);
             assert_eq!(fw.process_inbound_packet(&make_peer(), &make_tcp(them, us, TcpFlags::FIN)), true);
+            assert_eq!(fw.tcp.lock().unwrap().len(), 1);
```

Comment:
The other end can never send a `ACK` but some other packets and I think that this will prevent us from ever expiring the cache entry (since any accepted packet right now bumps the timeout by ttl).

I'm also to be honest not sure what to do with such packets, should be change current bevaiour and drop them?

If we want to handle that, this might require us to switch from `get` to `peek` and bump the timeout only if we are not waiting for closing `ACK`.

Reply:
>>> I'm also to be honest not sure what to do with such packets, should be change current bevaiour and drop them?

idk actually, redid so it drops them now :shrug: